toolchain/lower/file_context.h

```diff
@@ -31,6 +34,22 @@ class FileContext {
     int32_t column_number;
   };
 
+  // Describes a specific function's body fingerprint.
+  struct SpecificFunctionFingerprint {
+    // Fingerprint with all specific-dependent instructions, except specific
+    // calls. This is built by the `FunctionContext` while lowering each
+    // instruction in the definition of a specific function.
+    // TODO: this can be merged with the function type fingerprint, for a
```

Comment:
```suggestion
    // TODO: This can be merged with the function type fingerprint, for a
```

[Style](https://google.github.io/styleguide/cppguide.html#TODO_Comments): We've been capitalizing the first word in TODOs.